frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -28,18 +25,32 @@ const StyledButtonSection = styled.div`
     margin-right: 0;
     gap: 10px;
 `
-const StyledFormSection = styled.div`
+const StyledFormContainer = styled.div`
     display: flex;
-    margin-left: auto;
-    margin-right: 0;
-    gap: 10px;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    flex-direction: row;
+    align: left;
+    align-items: flex-start;
+    align-content: flex-start;
+    width: 50%;
```

Comment:
When the screen is slim (iPad mode) the table becomes  very small and wraps even if there is more room